composer.json

```diff
@@ -38,6 +38,9 @@
         "zendframework/zend-servicemanager": "~2.2",
         "zendframework/zend-view": "~2.2"
     },
+    "conflict": {
+        "twig/twig": "<1.38.1||<2.12.1"
```

Comment:
This ensures that the library is not installed along with an incompatible Twig version.